source/guides/account/scams-and-security.rst

```diff
@@ -0,0 +1,94 @@
+.. post:: 24 Feb, 2022
+    :category: Account
+    :tags: wallet
+    :excerpt: 1
+    :nocomments:
+
+######################
+Common Hacks and Scams
+######################
+
+***********
+Key Points
+***********
+
+- **Be extremely cautious signing transactions when activating delegated harvesting**. A Sweeping scam has actively targeted users in the process of activating delegated harvesting, so make sure to carefully review any transactions you are prompted to sign and diligently check for any included asset transfers to unfamiliar addresses. Never sign a transactions that you do not completely understand.
+- **Never share your private key or mnemonic phrase** (also known as your secret recovery phrase or seed phrase) with anyone, and never enter them on any website or potentially malicious wallet. Anyone with your private key or mnemonic phrase has full and unrestrained access to the contents of your wallet.  
+- **The Symbol Syndicate will never directly message you for customer support** and will never ask for your private key or mnemonic phrase. 
+- **Blockchain transactions are irreversible**; it is not feasible to undo a transaction or retrieve your funds once they are sent to a malicious address. 
+- **Ensure any backups of your private keys or mnemonic phrases are stored safely and offline** in order to ensure that no one is able to access them and compromise your accounts.
+
+*******************************************
+Sweeping with Aggregate Bonded Transactions
+*******************************************
+
+**What is an Aggregate Bonded transaction?**
```

Comment:
This is a "fake header". I prefer using proper headers so that the page structure is shown on the TOC and is easier to navigate. Also, it ensures homogeneity across the docs, and it's easier to maintain.

It should be:

```rst
What is an Aggregate Bonded transaction?
========================================
```

Reply:
Clarification request - are you recommending that _all_ of these fake headers are converted to headers (i.e. rather than just adding a space per your other 2 comments, also convert them to proper headers)?

Reply:
Yes, all fake headers should be turned into proper headers. Don't use fake headers anywhere.
AND also add the missing blank lines.

Reply:
Replaced all fake headers as requested